lua/ide/components/changes/component.lua

```diff
@@ -368,7 +368,7 @@ ChangesComponent.new = function(name, config)
         return commands.new(self).get()
     end
 
-    vim.api.nvim_create_autocmd({ "BufEnter", "CursorHold" },
+    vim.api.nvim_create_autocmd({ "BufEnter", "TextChanged", "InsertLeave" },
```

Comment:
Hey there. I originally had this as text changed but i remeber it causing some issues. Let me see if i still can determine why

Reply:
`TextChanged` only fires from normal mode, you can include insert mode with `TextChangedI`

Reply:
No it was a different issue, doing the lines with the plugin itself. I'll test